tasks/gulp-build.js

```diff
@@ -17,6 +17,6 @@ gulp.task('build:dev:watch', ['build:dev'], () => {
   gulp.watch(['website/public/**/*.styl', 'common/script/*']);
 });
 
-gulp.task('build:prod', ['browserify', 'prepare:staticNewStuff'], (done) => {
+gulp.task('build:prod', ['browserify', 'prepare:staticNewStuff', 'apidoc'], (done) => {
```

Comment:
I don't have a strong preference about this, as long as it's fast as it currently is we can also add it